sdk/storage/Azure.Storage.Blobs/src/BlockBlobClient.cs

```diff
@@ -2160,6 +2160,195 @@ protected sealed override BlobBaseClient WithSnapshotCore(string snapshot)
         }
         #endregion OpenWrite
 
+        #region UploadFromUri
+        /// <summary>
+        /// The Put Blob from URL operation creates a new Block Blob where the contents of the
+        /// blob are read from a given URL.  This API is supported beginning with the 2020-04-08 version.
+        ///
+        /// Partial updates are not supported with Put Blob from URL; the content of an existing blob is
+        /// overwritten with the content of the new blob.To perform partial updates to a block blob’s
+        /// contents using a source URL, use the Put Block from URL API in conjunction with Put Block List.
+        /// </summary>
+        /// <param name="copySource">
+        /// Required.  Specifies the URL of the source blob.  he source blob may be of any type,
```

Comment:
NIT - ..The* source blob.....

Reply:
Also looks like this was copied and pasted to the other overload comments.

Reply:
What should we change the wording to?

Reply:
Fixed!